mxnet/training/docker/1.6.0/py3/Dockerfile.cpu

```diff
@@ -105,7 +105,7 @@ RUN ${PIP} install --no-cache --upgrade \
     onnx==1.6.0 \
     numpy==1.17.2 \
     pandas==0.25.1 \
-    Pillow==6.2.0 \
+    Pillow==6.2.2 \
```

Comment:
can we update this to 7.1.2 for all py3 images?

Reply:
+1.

Reply:
We could try it. Note that some of these are old images that I'm not sure we will release again, so I wanted to keep the major version the same.

@Satish615 if I update, do you want me to rebuild all the old containers and test, or do you think it's safe to just test the new containers?